src/app/components/bootstrap.tsx

```diff
@@ -122,6 +123,43 @@ const Bootsrap: React.FC = () => {
       useSessionStore.getState().setFrequency(199998000);
     });
 
+    // Received when a station's state is updated externally, typically
+    // by another client via a websocket message. When received go through
+    // and ensure the state of the button in TrackAudio matches the new
+    // state in AFV.
+    window.api.on("station-state-update", (data: string) => {
```

Comment:
Handles updating the UI when things change via a websocket.

Given my earlier "more on that later" comment, this also gets triggered when TrackAudio UI changes something. Technically that's an unnecessary update since the radio UI code does its own state update but... that could all be removed and instead the UI could just update based on what the backend tells it is the state.